re #889 Update Naming

kahboom opened the pull request:
cc @EricWittmann - Still working on the last 7 blog posts.

EricWittmann commented:
OK cool - thanks @kahboom  - should I wait to merge until those last 7 are done?

kahboom commented:
@EricWittmann - Up to you, I'm guess it will take me about an hour or two to look over everything, make sure it looks okay, and maybe have time to edit the images. Right now I only need about 4 more posts (no images).

EricWittmann commented:
OK great - I'll wait until you've got the rest ready I guess. :)

msavy commented on _blog-src/_layouts/post.html:
```diff
@@ -11,6 +11,21 @@ <h1 style="color: #666; font-weight: bold" class="post-title section-header">{{
+
+    {% if page.oldUrl %}
+    <!-- New Post -->
+    <p>NOTE: This is a redux of our popular Keycloak OAuth2 for *apiman 1.2.x*. If you're still using apiman 1.1.x, you can refer to the <a href="{{ site.url }}{{ site.baseurl }}/{{ page.oldUrl }}">older revision</a>.</p>
```
> NOTE: This is a redux of our popular Keycloak OAuth2 

I guess this is artefact when you copied over the 'old/new blog post' notices from the KC blogpost. Presume you'll want to make it more generic. In asciidoctor `NOTE:` is an admonition, and it doesn't work in markdown, so suggest you kill that bit.

kahboom replied:
haha, good catch. I'm sure that'd confuse quite a few people.

kahboom replied:
@msavy - Maybe something like this:

> Note: This is a redux of an existing post for apiman 1.2.x. If you're still using apiman 1.1.x, you can refer to the older revision.

msavy replied:
How’s about:

This is a redux of an existing blogpost for apiman 1.2.x […]

Are you redoing all of them or just popular ones? If only popular ones I vote s/an existing/a popular/

> On 11 Jan 2016, at 21:15, Rachel Yordán <notifications@github.com> wrote:
> 
> In _blog-src/_layouts/post.html <https://github.com/apiman/apiman.github.io/pull/38#discussion_r49379648>:
> 
> > @@ -11,6 +11,21 @@ <h1 style="color: #666; font-weight: bold" class="post-title section-header">{{
> >      <span style="color: #243446; margin-left: 15px; font-size: 13px"><i class="fa fa-tags"></i> {{ page.categories | array_to_sentence_string }}</span>
> >    </header>
> >  
> > +
> > +  {% if page.newUrl || page.oldUrl %}
> > +  <div class="name-change-alert" style="margin-top:15px;">
> > +    {% if page.newUrl %}
> > +    <!-- Old Post -->
> > +    <p>NOTE: A newer revision of this blogpost correct for <a href="{{ site.url }}{{ site.baseurl }}/{{ page.newUrl }}">apiman 1.2.x is available</a>. This version written for apiman 1.1.x, for which it remains valid.</p>
> > +    {% endif %}
> > +
> > +    {% if page.oldUrl %}
> > +    <!-- New Post -->
> > +    <p>NOTE: This is a redux of our popular Keycloak OAuth2 for *apiman 1.2.x*. If you're still using apiman 1.1.x, you can refer to the <a href="{{ site.url }}{{ site.baseurl }}/{{ page.oldUrl }}">older revision</a>.</p>
> @msavy <https://github.com/msavy> - Maybe something like this:
> 
> Note: This is a redux of an existing post for apiman 1.2.x. If you're still using apiman 1.1.x, you can refer to the older revision.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/apiman/apiman.github.io/pull/38/files#r49379648>.
>

kahboom replied:
@msavy - All of them since there were only about 13 of them.

kahboom commented:
@EricWittmann - I pushed up my changes yesterday, which covers all blog posts, however, I'd like for you to take a look at the `2015-05-19-rest-api-redux.markdown` post to make sure everything was updated properly, as I felt the least confident with those changes. I'm just editing some images for the blog posts now and will do a sweep through the blog in case I broke anything.

kahboom commented:
<img width="1435" alt="screen shot 2016-01-12 at 9 44 50 am" src="https://cloud.githubusercontent.com/assets/3844502/12266439/27f6b6f6-b911-11e5-9a86-569fa780e9d1.png">

Fixed the styling, updated the text (see screenshot), and fixed an issue where the newer posts were not displaying the notice block. Working on images and also, I'm not sure what we want to do about duplicate entries in the list of blog posts, if anything at all, as that may be confusing for users.
